GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1151,6 +1151,20 @@ private void UpdateStashCount()
             }
         }
 
+        public override IScriptOptionsProvider? GetScriptOptionsProvider()
+        {
+            if (fileTree.Visible)
+            {
+                return fileTree.GetScriptOptionsProvider();
+            }
+            else if (revisionDiff.Visible)
+            {
+                return revisionDiff.GetScriptOptionsProvider();
+            }
+
+            return base.GetScriptOptionsProvider();
```

Comment:
```suggestion
            return revisionDiff.Visible ? revisionDiff.ScriptOptionsProvider
                : fileTree.Visible ? fileTree.ScriptOptionsProvider
                : base.GetScriptOptionsProvider();
```

Reply:
If it's OK I would like to avoid a nested ternary, at least for me it consumes way too many 🧠 cycles each time determining the order or evaluation and understanding the "what does this mean" without explicit parentheses.